### PR TITLE
cloudfoundry_v3_app resource experiment

### PR DESCRIPTION
**Pull request by chrisfarms**
Experimental cloudfoundry_v3_app resource using v3 api for rolling deployments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,110 +1,10 @@
-# Cloud Foundry Terraform Provider [![Build Status](https://travis-ci.org/cloudfoundry-community/terraform-provider-cloudfoundry.svg?branch=master)](https://travis-ci.org/cloudfoundry-community/terraform-provider-cloudfoundry)
+# Cloud Foundry Terraform Provider (v3 API)
 
+Experimental implementation of terraform resource for rolling deployments of
+cloudfoundry applications using the v3 API.
 
-Overview
---------
+Long-term intension is to contribute back
```
Comment (risicle) [inline README.md]: s/intension/intention/

**Review by risicle — APPROVED** [review]
Quite some experiment.

I hope nobody's tracking this repo's `master`...

File: README.md (hunk 1 above)
Comment (chrisfarms) [reply README.md]: stupid non-phonetic language... fixed

Comment (risicle) [reply README.md]: It'll be fine when we all switch to korean

**Pull request merged**